mmv1/products/dialogflowcx/terraform.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  Agent: !ruby/object:Overrides::Terraform::ResourceOverride
+    timeouts: !ruby/object:Api::Timeouts
+      insert_minutes: 40
+      update_minutes: 40
+    examples:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "dialogflowcx_agent_full"
+        primary_resource_id: "full_agent"
+        vars:
+          agent_name: "dialogflowcx-agent"
+    id_format: "{{name}}"
+    import_format: ["projects/{{project}}/locations/{{location}}/agents/{{name}}"]
```

Comment:
This will need to be `"{{name}}`, matching the `id` and `self_link`. As-is, the following snippet occurs:

```
2021/06/03 19:12:26 [DEBUG] Retry Transport: Stopping retries, last request was successful
2021/06/03 19:12:26 [DEBUG] Retry Transport: Returning after 1 attempts
2021/06/03 19:12:27 [DEBUG] matching ID projects/ci-test-project-188019/locations/global/agents/85cae1dd-8f17-4782-a302-8cbbef7f4b13 to regex projects/(?P<project>[^/]+)/locations/(?P<location>[^/]+)/agents/(?P<name>[^/]+).
2021/06/03 19:12:27 [DEBUG] importing project = ci-test-project-188019
2021/06/03 19:12:27 [DEBUG] importing location = global
2021/06/03 19:12:27 [DEBUG] importing name = 85cae1dd-8f17-4782-a302-8cbbef7f4b13
2021/06/03 19:12:27 [DEBUG] Waiting for state to become: [success]
2021/06/03 19:12:27 [DEBUG] Retry Transport: starting RoundTrip retry loop
2021/06/03 19:12:27 [DEBUG] Retry Transport: request attempt 0
2021/06/03 19:12:27 [DEBUG] Google API Request Details:
---[ REQUEST ]---------------------------------------
GET /v3/85cae1dd-8f17-4782-a302-8cbbef7f4b13?alt=json HTTP/1.1
Host: global-dialogflow.googleapis.com
User-Agent: Terraform/0.14.8 (+https://www.terraform.io) Terraform-Plugin-SDK/2.5.0 terraform-provider-google-beta/dev
Content-Type: application/json
Accept-Encoding: gzip
```

Reply:
Ok I changed it